src/components/shoppingListItem/shoppingListItem.module.css

```diff
@@ -39,22 +58,11 @@
   overflow-wrap: break-word;
 }
 
-.descriptionContainer {
-  display: grid;
-  border-right: 1px solid var(--border-color);
-  align-items: center;
-}
-
-.descriptionContainerEditable {
-  display: grid;
-  grid-template-columns: 25% 75%;
-  align-items: center;
-}
-
 .editIcons {
-  display: flex;
-  flex-direction: row;
-  padding: 4px 8px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  padding: 0 8px 4px;
+  margin-top: -2px;
```

Comment:
Vertical alignment of the `editIcons` seemed off because of the way the `faPenInSquare` icon's pen extends above the square. These styles position the icons a little higher, making them appear more centred.